### DISCUSSION TITLE
url must be a string

### DISCUSSION THREAD

**Discussion by dawnsongpeng** [umijs/umi#8970] [General]
您好，我在使用umi时出现了 url must be a string 的前台报错，请问该如何解决该问题？我在其他的网址看到说是useRequest的问题，但是，我将我所用到的useRequest去掉，依然汇报这个错误。谢谢你，我将不胜感激！！！

**Comment by fz6m** [comment]
由于缺乏足够的信息，请提供 [最小重现](https://stackoverflow.com/help/minimal-reproducible-example) 的 demo 帮助排查问题。

**Reply by dawnsongpeng** [reply]
非常感谢，已经找到出错的原因了，是我的问题，疏忽了相关的request请求参数。
